src/main/java/uk/gov/hmcts/reform/waworkflowapi/clients/model/CamundaProcess.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.hmcts.reform.waworkflowapi.clients.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@EqualsAndHashCode
+@ToString
+@Builder(toBuilder = true)
+public class CamundaProcess {
+    private final String id;
+    private final String definitionId;
+    private final String businessKey;
+    private final String caseInstanceId;
+    private final Boolean suspended;
+    private final Boolean ended;
+    private final String tenantId;
+
+    @JsonCreator
+    public CamundaProcess(@JsonProperty("id") String id,
```

Comment:
I presume by default its camelCase for Camunda , if it is then we don't ned @JsonProperty to set it explicitly.

Reply:
done